enterprise/libs.jackson/manifest.mf

```diff
@@ -2,4 +2,5 @@ Manifest-Version: 1.0
 OpenIDE-Module: org.netbeans.libs.jackson/2
 OpenIDE-Module-Localizing-Bundle: org/netbeans/libs/jackson/Bundle.properties
 OpenIDE-Module-Specification-Version: 2.24
```

Comment:
I'd bump the spec version since the lib now contains additional feature. If done, update depended-on version in the modules that use jackson/yml as well.